Jenkinsfile

```diff
@@ -33,6 +33,7 @@ pipeline {
 
         stage('Build') {
             steps {
+                sh "make verify-latest-onprem-config"
```

Comment:
This doesn't really relate to any of the other things we're doing in this job. Do we really want this is a step here?
Maybe it should be a separate stage? @YuviGold @oshercc ?

Reply:
@sacharya 
I believe we can add the new check under already existing `ci-lint`